comps/comps-foreman-client-el8.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<!DOCTYPE comps PUBLIC "-//Red Hat, Inc.//DTD Comps info//EN" "comps.dtd">
+<comps>
+  <!--  <meta> -->
+  <!-- Meta information will go here eventually -->
+  <!--  </meta> -->
+  <group>
+    <id>foreman-client</id>
+    <name>Foreman Client</name>
+    <description>Foreman Client Packages</description>
+    <uservisible>true</uservisible>
+    <packagelist>
+      <packagereq type="default">ansible-collection-theforeman-foreman</packagereq>
+      <packagereq type="default">foreman-client-release</packagereq>
+      <packagereq type="default">katello-agent</packagereq>
+      <packagereq type="default">katello-host-tools</packagereq>
+      <packagereq type="default">katello-host-tools-fact-plugin</packagereq>
+      <packagereq type="default">katello-host-tools-tracer</packagereq>
+      <packagereq type="default">python2-apypie</packagereq>
```

Comment:
Do we want python2 packages on EL8?

Reply:
:woman_shrugging: (no, but it's also not built there)